lib/web_ui/lib/src/engine/compositor/layer_scene_builder.dart

```diff
@@ -124,6 +124,13 @@ class LayerSceneBuilder implements ui.SceneBuilder {
     throw UnimplementedError();
   }
 
+  @override
+  ui.ImageFilterEngineLayer pushImageFilter(ui.ImageFilter filter,
+                                            {ui.ImageFilterEngineLayer oldLayer}) {
+    assert(filter != null);
+    throw UnimplementedError();
```

Comment:
Please, file an issue to implement it on the web. We can take it from there.

Reply:
Done: https://github.com/flutter/flutter/issues/47163